### DISCUSSION TITLE
你484不想上網課?

### DISCUSSION THREAD

**Discussion by henrywen547** [1101-GS4719-C/Python-Community-GS4719-C#629] [優秀專案]
有簡單的python程式碼幫你登入登出 google meet !
# [automeetit](https://github.com/ayushhsinghh/automeetit)
雖然這是一個比較少關注的一個專案(其實還有蠻多類似的作品)
但是作者本人有錄影片放youtube教你怎麼用
[Automatically Join Google Meet On a Given Time | Python Automation](https://www.youtube.com/watch?v=gNwRXNlk10c)

# 使用方法
## **Step 1**
下載.zip檔 後解壓縮
或你要git clone 也行啦
## **Step 2**
在資料夾中開啟終端機 然後
`pip install -r requirements.txt`
## **Step 3**
執行main.py
## **Step 4**
依照指示輸入內容
**記得會議連結要刪除"-"**
## **Step 5**
讀懂程式碼
_____________________________________
.
.
.
.
.
.
_**有些老師上課會問問題，不能用謝謝。請專心上課謝謝。**_

**Comment by gnsJhenJie** [comment]
https://chrome.google.com/webstore/detail/google-meet-scheduler-aut/gknbkfmgpeonofbofpgkngcoccblohgg
也可以直接用這個瀏覽器外掛XD

**Comment by Yao900108** [comment]
難怪老師不用google meet

**Comment by minniewei** [comment]
意外收穫了一個...有用的東東(誤)

**Comment by awkward-willy** [comment]
大家..還是...要好好上課...
現在全面網課的情況感覺會讓整體知識水平大幅下降
畢竟大家都在螢幕後面耍廢 XD

**Comment by Jay-722** [comment]
哈哈，這種東西也有，但還是認真上課

**Comment by 11timyyy** [comment]
面對上課會問問題的老師直接被gank，還是認真上課當乖學生XDD

**Comment by d26488399** [comment]
如果突然出現'我不是機器人'就尷尬了

**Comment by Lai27330** [comment]
好有趣，居然還有這一招，但很好奇有些時候掛著掛著就自動跳出的問題也能避免嗎?

**Comment by HsuChen88** [comment]
適合壞份子 但Python課我們都很認真上 所以才會用這個程式😎

**Comment by jason79461385** [comment]
好猛，壞份子上課打ㄟ配最適合用這個了

**Comment by SamTU0222** [comment]
謝謝分享 這可能也是大部分老師不想進行碗端上課的理由之一吧 哈哈

**Comment by yenwei12138** [comment]
感覺......好像不夠實用ㄟ，大家還是乖乖聽課為妙

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by etin2468** [comment]
感覺下學期就用不到了

**Comment by Ben6uo** [comment]
太爽了 感謝分享 那樣就可以偷懶了

**Comment by Lynn12378** [comment]
很好用的感覺，但是還是要好好上課，不然期中期末真的會沒救QQ

**Comment by 110409538** [comment]
但是之後很少會有線上上課的機會

**Comment by Jimmy911218** [comment]
==謝謝簍主提供喔

**Comment by Hchejxnc** [comment]
沒想到還有這個東西可以用，但習慣了期中期末就要完蛋了..

**Comment by orianna901001** [comment]
感覺是個有用的東西，謝謝版主的分享。
但超可惜的是，我們系上課都是小班制，所以常常會點名回答(再寫一個幫我回答問題的project吧QQ)

**Comment by Jamesmoneya** [comment]
還是好好上課啦，老師其實都知道，所以如果他一個突襲點名或是問問題，那就真的GG了，還是乖乖打開電腦聽老師在講甚麼，也能回答會提加分
